examples/mallocMC_example01.cpp

```diff
@@ -78,14 +93,20 @@ ALPAKA_STATIC_ACC_MEM_GLOBAL int** arC;
 
 auto main() -> int
 {
-    constexpr auto block = 32;
-    constexpr auto grid = 32;
     constexpr auto length = 100;
-    static_assert(length <= block * grid, ""); // necessary for used algorithm
 
     const auto dev = alpaka::getDevByIdx<Acc>(0);
     auto queue = alpaka::Queue<Acc, alpaka::Blocking>{dev};
 
+    auto const devProps = alpaka::getAccDevProps<Acc>(dev);
+    unsigned block = std::min(static_cast<size_t>(32u), static_cast<size_t>(devProps.m_blockThreadCountMax));
+
+    std::cout<< block<<std::endl;
```

Comment:
Is this correctly formatted?

Reply:
well spotted, it is a leftover from my tests